src/test/lit-element_test.ts

```diff
@@ -266,4 +269,32 @@ suite('LitElement', () => {
         assert.equal(a.foo, 20);
         assert.equal(a.shadowRoot!.textContent, '20');
       });
+
+  test(
+      'if `render` is unimplemented, do not overwrite renderRoot', async () => {
+        class A extends LitElement {
+          addedDom: HTMLElement|null = null;
+          createRenderRoot() {
+            return this;
+          }
+          connectedCallback() {
```

Comment:
This test works, but I think the connected callback is unnecessary. The test code can just assert that the child is still present after the update.

Reply:
Done